### ISSUE TITLE
Add support for aggregate emails

### ISSUE THREAD

**Issue by pippinsplugins** [AffiliateWP/AffiliateWP#1485]
Imagine an affiliate that earns a significant number of referral commissions, perhaps through 50 or a 100 subscriptions connected to Recurring Referrals. In this scenario, getting 50-100 emails (one for each commission) is annoying, especially if it happens every month when those subscriptions renew.

A great idea was proposed to me today to add an aggregate email that, perhaps once per week or month, sends affiliates a report of all commissions earned since the previous email.

**Comment by DrewAPicture** [comment]
Interesting. I think it might also serve as a subtle recurring reminder to affiliates of their activity on a weekly (or whatever) basis.

**Comment by pippinsplugins** [comment]
Agreed. We could show data on referrals, visits, and conversion rates.
